ui/app/AppLayouts/Chat/CommunityComponents/CommunityProfilePopupOverview.qml

```diff
@@ -232,10 +235,13 @@ Item {
             //% "Leave community"
             label: qsTrId("leave-community")
             iconName: "leave"
+            type: globalSettings.theme === Universal.Dark ? "secondary" : "primary"
         }
 
         Loader {
-            active: isAdmin
+            // TODO(pascal): reactivate with `isAdmin` once delete functionality
+            // is actually implemented
+            active: false
```

Comment:
I don't think delete will ever be implemented. So we can just remove it. If we ever add delete, it's not gonna be hard to re-add this any way